<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VcsBase::Internal::CheckoutProgressWizardPage</class>
 <widget class="QWizardPage" name="VcsBase::Internal::CheckoutProgressWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="logPlainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
